notebooks/analysis/MCMICRO_with_IDC.ipynb

```diff
@@ -0,0 +1,3664 @@
5F4VFHI0KeoGdChq8WiwWdXo9epyPSXMbq33jcLGHqPRF2jrxxJs1zSCaTzH10hWRbNfpNddh2NhKPx0XF7OIidXV1lL60S3oMg0NDOJ1Oqj63P01AybPoIRgIYjKbCASCJFoqsVgtgkXTUon/TBfGW8Pc7LpF3pHNFBYWYrFaCYVCGJ7wqvU3BtAb9GzavAkA49YGBl49Qd7TW0RSMBjko3/1Z/S/epyRN84InPpAG4mOh/J+d3/rLWKxGIXFRWQdWo9rf2vahpf2fXq9SD7arCjXRfFS9FIPE+9ckO9RVZXw8iYai8ex3hmTJftt3/icMEJWqzDwbULONdPtBkREEl1bjs1qFRvNpjrsu5spfG4bpSUlMpyPx+MMvHpCnuf0B5fR6/V4FxdxP5oVeYD/8irVnz9AsuMhgUCQpaVFlpaWKJ+KETp3h0QyQWxtuZQVyD2yWYTry9FLKkylQRxa4k+/uR7PopfhEcHOYv0KvB4PibYqEskkq7/+3GNRrM312Hc3oygK1p2NlLy4S2LReXl5GI1Gar98WEg7Wy3oN9dT+tIu7Bl2Rt88y9Drpwid707byIqe2459dzP1XznyeN6+f0mcY8dDMpY90on3LsrNKi8vjz379mHYspJkMsnD73+Iqoqajrm5Oe5++x2sOxsxGI1EIhHi7Q+IXOoh6+B6XC4XBSMiR6LlASoqKkTDlfYHUuI4kUgw99EVwXJpv8/0+6L7knFrA7mHBf89eeUR0Uv3sO9uAoSBHH9XqEkmrzxi0Sd6DQSDQbwnrzP7UQeL9Xk4nU4pnKd29jJ/tJPMA23AYykRrbbEta9FRqDx9gckrzySz1JRFBaOX2Xu6BXCF3u4d+8ew8trXXdjkHjHw0+0Oanjk2o3PokbL3//rxWi+fvGwYMH+d73vsfp06f59//+33P+/HkOHTokca2pqSny8vLSPmMwGMjKymJqakq+R2sErQ3td+09T44//MM/ZHFxUf4bHV0WVDrfTSgUEgU1rZWCwTA7w9Drpwgu65eACLu9Xi/Bc91YbVaSqlDTM25rEIqN68rlA9OiE82wVFZWoHb2kt23IMrZ3z5P0Qs78C56mZqaxri1AdO2VdTW12K1Wun/0XFsNhuLi4vMz89TWFCAqsDkuxcxdQ1jvDWM7/Qt0SDj2FX8fj+5RzbhyHCwePIGCwsLxC7dw3+mC8/xa/IeKIooapl67xLX//LH2HY14Xa7CQaDlJaVCh76skiTZqD0ej31K+vxNRTg2LtOTkT95vplyV0zOTk5aR2eNEOlGaVYLIbVYqGhoYFIJCKSpLub5IKyWCzQudwt66qQEZ75oF3w8P/yVWE84nFBCbzSu7xJ3JPfF7t0j8VTN8R5XR8gHo/j9wcYK7WRSCZIrq/G7/eneW7a9SU6HmDevhrrzjUUPb+DQCDA+t//Aoqi0PejYxhvDXPbGiAzMxOn00l0mWWhldUnWirl75qWj3aPvF6vNIjbd2yn7wdHAcg82Iapa4ScI5tEgnG9qA7NPbKJnMObSHQ8JHCmi9i6CgDZRCS5vprxiXFil+9z7564/vjl+wA8fPQIm90mvW23252WpJ145wJLJ28w+sZZSl7chXn7ankfgsGgPI7GiAGYnp7m/B9/S74m+vZmCPho71qB8V/rR7naJze3wJoiRpe7b83NzWHYulIa07mPrlBQUEBWVhbZ2dlYrVaqPn8Ao9GIw+EQOLnTKRyeK4+IxePE1ok8g3PfOgJnhER33fImpapC9tfpdKBuWCHPX0x2EXl7jl8j+6kN0oP2nriOZcca9JvrJWQYjUbxnb6VVs2r6emkRk8Ze0TFbeBc98/Ndkl9Jp8GyfwyeS2/8EKnz372s/LnNWvW0NjYSHV1NefOnWPPnj2/6K+Tw2w2p1HB5NASW7sasdtsLG4143a58GyuI3pBwC36LfVyYsdMM0Q6+3jYIWRdW1pauHtrhIZ1a7l7U+DUFouFRFsVTU2N3P3WO9y4eZOy7Gxm5+Zkwc6D77z/uDgIsOxYw+CrJ8XiPNiG8cYQFouFmZkZXMvnvaA+bkxg390sj+V0OpmbmycQCpK7dSUGg4GF5Ymrpnwm98gmkqqK5+hVXPtbmHj7Av7lyXvtz38kWBbPbGHotVMC/9xST/jcHS7+ybcxJhIsKMPUfvkwgz8+QfRiD0Ob45hvDkvWDTzmxGshtnaNpZMRom2VxKxBcnNy8Pv8ZB5aj8fjwWS3o1MU5k/cSJvcqdi2qqqYd64RG8YF0R9Uk2kGYGON0Bw5eZPouW4yn99Oov0Bht1NFOTnM7h8zOT6ajIzM5mfn8d8a4R4ayXjb50TU2FzLaFQGKvdhu56P/nFxUxXuVh68xwlGzfiu/yAgg3VFH31ae5+620SyQR2RwYLCwvk7WokdvGevP54PI66YQWusjISCwt0XLnyWGTN7Sa61oZ30UtuXh6qAnf/9h1JATVubyB24R6GG4NgtaIzm8l5ehOL3kUCwSDxVQWsyMtl5OYw8Xic7Ew3M+9dJrpskEpKSlA3rMD7xll571RVaBupG2uY/aAd2/ZVeJY3g/zntjLz7mW5ORS+sJ3s7Gzu3bvHuo2bOHvuHOvXt3H7r99gYmKC/Oe2Yr4xzHAshrqxRgihXREbc0SBZMdDcn/3ZaJvnCOzoADTF8twud0k4nGm3hG6MplPrcfv83P/3n0Sy1IZqqpS9MIOPAsLzH7QgefYVWxGI8btDdjsdqaWI4v7f/ceFRUVLDXk4/F6iMzNY7w5hG6Zs67T6YhdFGJlK750iP6+fkjxoCPRCMn2h8vLX6H6CwcZHh5G1/GIwue2seDxEDrbjWt/CwtHRWFXNBqlwO2i9vde4c4330rrsfykJ/6zGOifhreLY/y9h/iFjH90HnxVVRU5OTn09YmO8wUFBR/L9MfjcRYWFqRnXFBQIDU2tKH9/mnY/qcNTRFSW1yJRIKkqpK4/EC+R5Mm0O56LBaTCcGO//Q9fD4f9779Hp
```

Comment:
I would recommend installing and checking all of the prerequisites in the beginning of the notebook.